src/allmydata/test/test_python3.py

```diff
@@ -100,8 +116,8 @@ def count_lines(module_name):
     try:
         source = module.filePath.getContent()
     except Exception as e:
-        print(module_name, e)
+        print((module_name, e))
         return 0
     lines = source.splitlines()
-    nonblank = filter(None, lines)
+    nonblank = [_f for _f in lines if _f]
```

Comment:
What's better, this pattern or `list(f for f in lines if f)`?  They're pretty close.  List comprehensions on Python 3 don't leak the loop variable, right?  So `_f` is to avoid name collisions/problems on Python 2 - but `list(...)` doesn't leak on 2 or 3.

Reply:
This is what futurize does automatically. So it has the benefit of being easy!

Plus `filter(None, x)` is a bit of an obscure idiom, so I prefer this version.